bpf/bpf_lxc.c

```diff
@@ -800,19 +818,23 @@ ipv6_policy(struct __sk_buff *skb, int ifindex, __u32 src_label, int *forwarding
 			return ret2;
 	}
 
-	verdict = policy_can_access_ingress(skb, src_label, tuple.dport,
-					    tuple.nexthdr, sizeof(tuple.saddr),
-					    &tuple.saddr, false);
+	if (!(cfg->flags & EP_F_SKIP_POLICY_INGRESS)) {
+		verdict = policy_can_access_ingress(skb, src_label, tuple.dport,
+				tuple.nexthdr, sizeof(tuple.saddr),
+				&tuple.saddr, false);
+	}
+	else {
+		verdict = 0 ;
+	}
 
 	/* Reply packets and related packets are allowed, all others must be
-	 * permitted by policy */
+         * permitted by policy */
 	if (ret != CT_REPLY && ret != CT_RELATED && verdict < 0) {
-		/* If the connection was previously known and packet is now
-		 * denied, remove the connection tracking entry */
-		if (ret == CT_ESTABLISHED)
-			ct_delete6(get_ct_map6(&tuple), &tuple, skb);
-
-		return verdict;
+        	/* If the connection was previously known and packet is now
+         	 * denied, remove the connection tracking entry */
+        	if (ret == CT_ESTABLISHED)
+            		ct_delete6(get_ct_map6(&tuple), &tuple, skb);
+			return verdict;
```

Comment:
nit: Whitespace is spaces here. (That's also why the `permitted by policy` line above reports a change)

Reply:
Additional nit, this messes up the indentation of the verdict return. The if statement deliberately only applies to the `ct_delete6()` line (no braces => only apply to the next statement), however all traffic that is rejected by the verdict and not reply / not related is dropped by this verdict return.

Is there any intentional change in these lines? Seems like the hunk could just be dropped.